### ISSUE TITLE
Language not load

### ISSUE THREAD

**Issue by PendalF89** [kartik-v/yii2-widget-datepicker#16]
Hi!
I'm using this extension and have some problem
```php
echo DatePicker::widget([
    'name' => 'check_issue_date',
    'value' => date('Y-m-d', strtotime('+2 days')),
    'options' => ['placeholder' => 'Select issue date ...'],
    'language' => 'ru',
    'pluginOptions' => [
        'format' => 'yyyy-mm-dd',
        'todayHighlight' => true
    ]
]);
```

But language not load! No any errors, just english language.
I found logical error in 140 line in DatePicker.php
```php
$this->setLanguage('bootstrap-datepicker.', __DIR__ . "/assets/");
```
so, add slash to assets:
```php
$this->setLanguage('bootstrap-datepicker.', __DIR__ . "/assets//");
```
and it works!
What you think about it?

**Comment by kartik-v** [comment]
I do not get you - why are you advising on **two forward slashes**?

**Comment by PendalF89** [comment]
![2015-01-25 18-57-19](https://cloud.githubusercontent.com/assets/5278851/5892063/315d1ae4-a4c4-11e4-96d5-4d5bb86c217b.png)

So, this will render this path: 'C:\\OpenServer\\domains\\365\\vendor\\kartik-v\\yii2-widget-datepicker/assetsjs\\locales\\bootstrap-datepicker.ru.js'

**assetsjs** !

**Comment by kartik-v** [comment]
The fix for this should be in yii2-krajee-base... will update.

**Comment by PendalF89** [comment]
Thank you!

**Comment by kartik-v** [comment]
This should be fixed via the commit here and the fix in kartik-v/yii2-krajee-base#27.

**Comment by PendalF89** [comment]
Great! Thank you so much!
